PUBLISHING.md

```diff
@@ -0,0 +1,219 @@
+git push origin code-assistant-v0.1.1
+
+# Create a GitHub release from this tag
+gh release create code-assistant-v0.1.1 --title "qiskit-code-assistant-mcp-server v0.1.1" --notes "Release notes here"
+```
+
+#### For Runtime Server:
+```bash
+# Update version in qiskit-ibm-runtime-mcp-server/pyproject.toml
+# Then create and push a tag
+git tag -a runtime-v0.1.1 -m "Release qiskit-ibm-runtime-mcp-server v0.1.1"
+git push origin runtime-v0.1.1
+
+# Create a GitHub release from this tag
+gh release create runtime-v0.1.1 --title "qiskit-ibm-runtime-mcp-server v0.1.1" --notes "Release notes here"
+```
+
+### Manual Workflow Trigger
+
+You can also trigger publishing manually via GitHub Actions:
+
+1. Go to **Actions** → **Publish to PyPI**
+2. Click **Run workflow**
+3. Select which package to publish: `both`, `code-assistant`, or `runtime`
+
+## Manual Publishing
+
+### Prerequisites
+
+Install build tools:
+```bash
+pip install build twine
+```
+
+Or use `uv` (recommended):
+```bash
+pip install uv
+```
+
+### Step-by-Step Manual Publishing
+
+#### 1. Update Version
+
+Edit the version in `pyproject.toml`:
+- **Code Assistant**: `qiskit-code-assistant-mcp-server/pyproject.toml`
+- **Runtime**: `qiskit-ibm-runtime-mcp-server/pyproject.toml`
+
+#### 2. Build the Package
+
+**For Code Assistant:**
+```bash
+cd qiskit-code-assistant-mcp-server
+
+# Build with uv (recommended)
+uv build
+
+# Or with build
+python -m build
+```
+
+**For Runtime:**
+```bash
+cd qiskit-ibm-runtime-mcp-server
+
+# Build with uv (recommended)
+uv build
+
+# Or with build
+python -m build
+```
+
+This creates `.whl` and `.tar.gz` files in the `dist/` directory.
+
+#### 3. Verify the Build
+
+Check the contents:
+```bash
+# List files in the wheel
+unzip -l dist/*.whl
+
+# Check package metadata
+twine check dist/*
+```
+
+#### 4. Upload to PyPI
+
+**Test on TestPyPI first (recommended):**
+```bash
+# Upload to TestPyPI
+twine upload --repository testpypi dist/*
+
+# Test installation
+pip install --index-url https://test.pypi.org/simple/ qiskit-code-assistant-mcp-server
+# or
+pip install --index-url https://test.pypi.org/simple/ qiskit-ibm-runtime-mcp-server
+```
+
+**Upload to production PyPI:**
+```bash
+# With twine
+twine upload dist/*
+
+# Or with uv
+uv publish
+```
+
+You'll be prompted for your PyPI username and password (or API token).
+
+#### 5. Verify Installation
+
+```bash
+# For Code Assistant
+pip install qiskit-code-assistant-mcp-server
+
+# For Runtime
+pip install qiskit-ibm-runtime-mcp-server
+```
+
+## Version Management
+
+### Versioning Strategy
+
+Both packages use **semantic versioning**: `MAJOR.MINOR.PATCH`
+
+- **MAJOR**: Breaking changes to the API
+- **MINOR**: New features, backward compatible
+- **PATCH**: Bug fixes, backward compatible
+
+### Current Versions
+
+- **qiskit-code-assistant-mcp-server**: `0.1.0` (Alpha)
+- **qiskit-ibm-runtime-mcp-server**: `0.1.0` (Alpha)
```

Comment:
do the current versions need to be listed here? if so, the instructions should include steps to update the values here so they remains up to date. otherwise, this could be removed or maybe be links to the version lines in the respective pyproject.toml